packages/dd-trace/src/ci-visibility/telemetry.js

```diff
@@ -0,0 +1,130 @@
+const telemetryMetrics = require('../telemetry/metrics')
+
+const ciVisibilityMetrics = telemetryMetrics.manager.namespace('civisibility')
```

Comment:
Is `civisibility` a valid namespace? 
At least in the docs it doesn't appear on the valid list ("tracers", "general", "telemetry", "iast", "appsec")

Reply:
Should be:
![image](https://github.com/DataDog/dd-trace-js/assets/136473744/ef7388b3-e3ae-4bc0-aaa5-35eeb5ac2dc9)